immudb/client.py

```diff
@@ -101,6 +101,9 @@ def stub(self):
     def healthCheck(self):
         return healthcheck.call(self.__stub, self.__rs)
 
+    def health(self):
```

Comment:
There could be a confusion between health and healthCheck calls - maybe we should deprecated one and prefer the other?

Reply:
Same in the Go SDK. Returned parameters are different.